doc/compodoc_sources/how-to-guides/review-pull-request.md

```diff
@@ -0,0 +1,98 @@
+3. Run the app locally.
+Put yourself in the shoes of a user and click around, testing the app:
+    1. Does the new feature / bugfix function as discussed and expected?
+    2. How are edge cases and errors handled?
+        * _This depends on the kind of change. If you can't think of any, just go on._
+    3. Does the user get feedback about what is going on in a way that is understandable for people with no technical knowledge?
+    (That is in case of success as well as failure.)
+    4. Do related features that might have been affected by the changes of this PR still work?
+        * _Again, this depends on the kind of PR. If you don't see anything else to test, that's absolutely okay._
+    5. If the change affects UI elements: How do they look in a smaller browser window?
+4. Write a comment in the PR on GitHub about what any problems or ideas you encountered.
+
+### (2.) Is the code tested with unit tests?
+The unit testing is checked automatically and concerns two aspects:
+* Are all tests passing?
+    * _That is new tests for this PR as well as all other tests to ensure no previously working function was broken._
+* Are the changes of this PR covered by unit tests?
+    * _We aim for a "test coverage" of at least 80% and PRs should contain new unit tests for the changes they are making._
+
+The CI system reports the status of these conditions automatically and displays them in the GitHub PR.
+
+You don't have to do any specific reviewing regarding this.
+If some tests fail, you may want to wait with your review until the author has fixed them so that you can review a final, working state of the code.
+Also, if you notice some important edge cases that are not tested by unit tests yet, do suggest these to be added.
+
+### (3.) Is the code understandable and well structured?
+Reviewing a PR means reviewing code.
+If you spot a logical error or have a suggestion how to rewrite some code to improve performance that's awesome.
+But even more important is asking the right questions and pointing out all the parts that are hard (or impossible) to understand.
+
+Rather sooner than later even the author of the code will have forgotten the hidden logic of that super complex function.
+And she/he will thank your for requesting a simpler code structure or a better documentation during your review.
+
+You can review the PR's code changes easily on GitHub:
+1. In the GitHub PR switch to the "Files changed" tab.
+2. Go through the code changes and comment on specific lines:
+    1. If you don't understand what a block of code does.
+    It should be possible to understand everything.
+    Be ruthless and ask about everything that's unclear - 
+    code structure and/or documentation should be improved for everybody in this case.
+    2. If you have a suggestion to improve the code.
+    Whether you have an idea to structure things in a clearer way
+    or to solve it differently - share your suggestions.
```

Comment:
```suggestion
    2. If you have a suggestion to improve the code.
    Whether you have an idea to structure things in a clearer way
    or to solve it differently - share your suggestions.
    3. You can also suggest code changes from within the GitHub UI that can be merged - without having the checkout and commit to the branch on you local machine. See [Suggesting Changes on GitHub](https://haacked.com/archive/2019/06/03/suggested-changes/)
```

Reply:
Is this just a random event, or did you plan the self-describing-feature of this comment? :D ( i wonder if this PR should not be merged as it is kind of a demo for pull requests, then?? - otherwise I'd approve and merge..

Reply:
haha, yeah. I read about the feature and tried it here
I love the meta layer of that, though ;-) 

Do approve and merge it. I think this should end up in the latest hosted developer docs.